bitmap-team-awesome/model/bitmap.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = function Bitmap(buffer){
```

Comment:
@meeksfred this totally works but the more common design pattern is to create your constructor definition like this:

`const Bitmap = module.exports = function(buffer)`